config/config.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	ShowONSMap                 bool
```

Comment:
Probably want to add in the envconfig tag to be able to pick an environment variable to effectively change ShowONSMap from false to true or vice versa.

We usually stick with the naming convention:

```suggestion
	EnableONSMap                 bool       `envconfig:"ENABLE_ONS_MAP"`
```

Default setting should be false

Reply:
Most of the map logic has been removed now, so this variables isn't being used, so i can just remove it.